### PR TITLE
Add few more Symfony aliases

### LINKED ISSUE #4: Add the following aliases for symfony
sft: php app/console --env=test

sftd: php app/console twig:debug
sfrd: php app/console router:debug
sfcd: php app/console container:debug

server:start
server:status
server:stop

doctrine:database:create
doctrine:database:drop
doctrine:fixtures:load
doctrine:generate:entities
doctrine:migrations:diff
doctrine:migrations:execute
doctrine:migrations:migrate
doctrine:migrations:status
doctrine:schema:create
doctrine:schema:drop
doctrine:schema:update
doctrine:schema:validate

cache:clear
cache:warmup

assets:install --symlink

### PR DESCRIPTION
**Pull request by calina-c**
- Closes #4

### CODE REVIEW

File: doc/bash/symfony_aliases.md [hunk 1]
```diff
@@ -40,4 +40,20 @@
 - **sfai**: `php app/console assets:install web --symlink`
 
 ### Symfony server ###
-- **sfs**: `php app/console se:ru`
+- **sfsr**: `php app/console server:run`
```
Comment (stocarul) [inline doc/bash/symfony_aliases.md:43]: Please indent this group of lines with one more space.

Comment (stocarul) [inline doc/bash/symfony_aliases.md:43]: Sorry. Not needed to do that.

File: doc/bash/symfony_aliases.md [hunk 2]
```diff
@@ -40,4 +40,20 @@
...
+- **sfss**: `php app/console server:status`
+- **sfsq**: `php app/console server:stop`
+
+### Doctrine ###
+docdbcreate=`doctrine:database:create`
```
Comment (stocarul) [inline doc/bash/symfony_aliases.md]: Please prefix all the lines from this group with `-` (dash followed by space), like in the above groups.

Comment (stocarul) [inline doc/bash/symfony_aliases.md]: And shouldn't this line be like this? (alias name prefixed with `sf` and the actual alias prefixed with `php app/console`)

```
sfdocdbcreate=`php app/console doctrine:database:create`
```

And the same for all the other aliases from this group.

**Comment by stocarul** [comment]
@calina-c: And also please use only `do` for doctrine in alias name instead of `doc`.

**Comment by calina-c** [comment]
Handled comments in new commit.

File: bash/symfony_aliases [hunk 3]
```diff
@@ -27,5 +27,21 @@ alias sfaw='php app/console assetic:watch'
...
 
-# Symfony server alias
-alias sfs='php app/console server:run'
+# Symfony server aliases
+alias sfsr='php app/console server:run'
+alias sfss='php app/console server:status'
+alias sfsq='php app/console server:stop'
+
+# Doctrine aliases
+alias dodbcreate='doctrine:database:create'
```
Comment (stocarul) [inline bash/symfony_aliases]: @calina-c: Shouldn't all the below alias's commands start with `php app/console`?

Comment (stocarul) [reply bash/symfony_aliases]: To keep everything consistent, wouldn't it be better to have all Symfony related aliases start with `sf`?

**Comment by calina-c** [comment]
@stocarul Fixed as instructed.

**Pull request merged**